src/Tizen.NUI/src/public/Utility/TextPageUtil.cs

```diff
@@ -27,318 +27,331 @@
 namespace Tizen.NUI.Utility
 {
 
-  /// <summary>
-  /// This is a class for stroing the text of a page.
-  /// </summary>
-  [EditorBrowsable(EditorBrowsableState.Never)]
-  class PageData
-  {
-    public string previousTag  {get; set;}
-    public string endTag {get; set;}
-    public int startOffset {get; set;}
-    public int endOffset {get; set;}
-  }
-
-  /// <summary>
-  /// This is a class that stores information when parsing markup text.
-  /// </summary>
-  [EditorBrowsable(EditorBrowsableState.Never)]
-  class TagData
-  {
-    public string tagName {get; set;}
-    public string attributeName {get; set;}
-    public bool isEndTag {get; set;}
-  }
-
-  /// <summary>
-  /// This is utility class for paging very long text.
-  /// </summary>
-  [EditorBrowsable(EditorBrowsableState.Never)]
-  public class TextPageUtil : Disposable
-  {
-    private static char LESS_THAN      = '<';
-    private static char GREATER_THAN   = '>';
-    private static char EQUAL          = '=';
-    private static char QUOTATION_MARK = '\'';
-    private static char SLASH          = '/';
-    // private static char BACK_SLASH     = '\\';
-    // private static char AMPERSAND      = '&';
-    // private static char HASH           = '#';
-    // private static char SEMI_COLON     = ';';
-    // private static char CHAR_ARRAY_END = '\0';
-    // private static char HEX_CODE       = 'x';
-    private static byte WHITE_SPACE    = 0x20;
-
-    private int totalPageCnt;
-
-    private List<PageData> pageList;
-    private List<TagData> tagList;
-    private StringReader stream;
-    private List<char> characterList;
-    private string pageString;
-
     /// <summary>
-    /// When text is inputed, the text is paging in the TextLabe size unit.
-    /// <returns>The total number of pages.</returns>
+    /// This is a class for stroing the text of a page.
     /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public int SetText(TextLabel label, string str)
+    class PageData
```

Comment:
this looks private class, but could you add private or internal or public explicitly?